external-crates/move/crates/move-compiler/tests/move_check_testsuite.rs

```diff
@@ -287,4 +287,4 @@ pub fn run_test_inner(
     }
 }
 
-datatest_stable::harness!(move_check_testsuite, "tests/", r".*\.move$");
+datatest_stable::harness!(move_check_testsuite, "tests/linter", r".*\.move$");
```

Comment:
Are you having trouble running your tests? It should already be picked up via the `LINTER_DIR` check

Reply:
i accidently changes this files. restored